t/re-bugs.t

```diff
@@ -137,3 +137,40 @@ c0
 disabled in init phase under macOS
 --- skip_eval
 4: $^O ne 'linux'
+
+
+
+=== TEST 5: bug: sub incorrectly dropped the last character
+--- config
+    location /re {
+        content_by_lua '
```

Comment:
Could you use content_by_lua_block style?

Reply:
done